bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/EnOceanHandlerFactory.java

```diff
@@ -60,6 +61,9 @@
     @Reference
     ItemChannelLinkRegistry itemChannelLinkRegistry;
 
+    @Reference
+    ThingManager thingManager;
```

Comment:
Can these be private?

Reply:
This dependency is injected by reflection, so it is no problem to add this annotation to private field. At least it works without a problem for me.